src/utils/is-bounding-box-ref.ts

```diff
@@ -0,0 +1,13 @@
+import { RefObject } from "react"
+import { BoundingBox2D } from "types/geometry"
+
+export const isBoundingBoxRefObject = (
+    ref: any
+): ref is RefObject<BoundingBox2D> => {
+    return (
+        typeof ref === "object" &&
+        ref.hasOwnProperty("current") &&
+        ref.current.hasOwnProperty("top") &&
+        ref.current.hasOwnProperty("left")
```

Comment:
Not suuuuuure if this is the best approach, if you can think of a better heuristic lmk